src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -448,29 +448,11 @@ private BoundStatement BindGoto(GotoStatementSyntax node, DiagnosticBag diagnost
 
         private BoundStatement BindLocalFunctionStatement(LocalFunctionStatementSyntax node, DiagnosticBag diagnostics)
         {
-            var binder = GetBinder(node);
```

Comment:
>var binder = GetBinder(node); [](start = 12, length = 29)

If we no longer need to locate the binder, we should probably change LocalBinderFactory to no longer add one into the map.